thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/models/AnomalyResultModel.js

```diff
@@ -111,8 +111,64 @@ AnomalyResultModel.prototype = {
   },
   updateModelAndNotifyView : function(anomaliesWrapper) {
     this.anomaliesWrapper = anomaliesWrapper;
+    this.formatAnomalies();
     this.renderViewEvent.notify();
   },
+
+  /**
+   * Helper Function that returns formatted anomaly region duration data for UI
+   * @param  [date] start: the anomaly region start
+   * @param  [date] end: the anomaly region end
+   * @return [string]  formatted start - end date/time
+   */
+  getRegionDuration(start, end) {
+    const regionStart = moment(start, constants.TIMESERIES_DATE_FORMAT);
+    const regionEnd = moment(end, constants.TIMESERIES_DATE_FORMAT);
+    const isSameDay = regionStart.isSame(regionEnd, 'day');
+    const timeDelta = regionEnd.diff(regionStart);
+    const regionDuration = moment.duration(timeDelta);
+    let regionStartFormat;
+    let regionEndFormat;
+
+    if (isSameDay) {
+      regionStartFormat = constants.DETAILS_DATE_DAYS_FORMAT;
+      regionEndFormat = constants.DETAILS_DATE_HOURS_FORMAT;
+    } else {
+      regionStartFormat = regionEndFormat = constants.DETAILS_DATE_DAYS_FORMAT;
+    }
+
+    return `${regionDuration.humanize()} (${regionStart.format(regionStartFormat)} - ${regionEnd.format(regionEndFormat)})`;
+  },
+
+  /**
+   * Helper Function that retuns formatted change delta for UI
+   * @param  [int] current: current average anomaly
+   * @param  [int] baseline: baseline the anomaly is compared too
+   * @return [string] 'N/A' if either is missing, otherwise formatted delta (%)
+   */
+  getChangeDelta(current, baseline) {
```

Comment:
You can skip the else by defaulting to `N/A` and checking only for the valid case:
````javascript
let changeDelta = 'N/A';

if (current && baseline) {
  const amount = (current - baseline) / baseline  * 100;
  changeDelta = `${amount.toFixed(2)}%`;
}

return changeDelta;
````

*Not a big deal if invocation is limited but you may also want to ensure that current and baseline can be parsed as valid numbers